public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx

```diff
@@ -81,9 +81,24 @@ export const placeholders = {
 
 export const DEFAULT_RESULTS_PER_PAGE = 10;
 
-export const MetricEncyclopediaModal = (props: Props) => {
-  const uf = UseUfuzzy();
+const uf = new uFuzzy({
+  intraMode: 1,
+  intraIns: 1,
+  intraSub: 1,
+  intraTrn: 1,
+  intraDel: 1,
+});
+
+function fuzzySearch(haystack: string[], query: string, setter: React.Dispatch<React.SetStateAction<number[]>>) {
+  // console.log('fuzzySearch');
```

Comment:
Ooops

Reply:
:facepalm: hehe, that was there to help @bohandley verify my debounce fix worked as expected. he's gonna be poking this whole file soon in https://github.com/grafana/grafana/issues/64577, so hopefully he catches it :)